extensions/amp-story/amp-story.md

```diff
@@ -395,6 +395,20 @@ This same image can be used for both mobile portrait and landscape desktop using
 <amp-img src="cat.jpg" alt="..." object-position="75% 40%"></amp-img>
 ```
 
+##### Optimize `amp-video` by using a video cache on origin documents
+
+The `<amp-video>` element on stories supports specifying a video cache to be used on origin documents through the attribute `cache`.
+The video cache can serve copies of the video at multiple bitrates, reducing the server's workload on delivering videos.
+The appropriate video quality plays on the story, according to the network conditions of the viewer.
+
+Example:
+
+```html
+<amp-video layout="fill" poster="img.png" cache="google">
+  <source src="video.mp4" type="video/mp4">
+</amp-video>
```

Comment:
Please copy the relevant parts of the existing external communication that has already been reviewed, so it's all consistent